github-management/opening-a-request.md

```diff
@@ -31,6 +31,7 @@ Please open an issue against the [kubernetes/test-infra] repository describing
 your issue. If your request is urgent, please escalate to the
 [test-infra on-call] or reach out to `#testing-ops` on Slack.
 
+_To report any sensitive information, please email the private github@kubernetes.io list_.
```

Comment:
Possible formatting suggestion/nit --  With this directly under the bot/automation header it kind of looks like its a part of that section. Maybe another header or a horizontal rule just to break it up?

Reply:
I thought it felt awkward too. Updated, ptal if this is any better!

Reply:
Thanks! That looks better.